### PR TITLE
Implemented RtRepositories.

### PR DESCRIPTION
**Pull request by SherifWaly**
PR for #89

### CODE REVIEW

**Comment by SherifWaly** [comment]
@amihaiemil There is a typo with the test class name so I will fix it first and I will try to apply filters

**Comment by SherifWaly** [comment]
@amihaiemil I fixed the type and added a puzzle as I'm not getting how to apply filters with mock testing

File: src/main/java/com/amihaiemil/versioneye/RtRepositories.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        );
+    }
+
+    @Override
+    public Page<Repository> paginated() {
+        return new RepositoriesPage(this);
+    }
+
+    @Override
+    public Repositories language(final String language) {
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/versioneye/RtRepositories.java:103]: @SherifWaly These methods look nice, but can you add a puzzle for: 

* a little refactoring, to make this class immutable.
* more unit tests.

**Review by amihaiemil — COMMENTED** [review]
@SherifWaly Looks nice :)

**Comment by amihaiemil** [comment]
@rultor merge pls

**Pull request merged**